src/components/text-editor/prosemirror-adapter/prosemirror-adapter.scss

```diff
@@ -13,10 +13,6 @@
         height: 100%;
         flex-grow: 1;
     }
-
-    div[contenteditable='true'] {
-        height: 100%;
-    }
```

Comment:
<img width="216" alt="Screenshot 2024-06-11 at 12 05 02" src="https://github.com/Lundalogik/lime-elements/assets/35954987/d8151ebf-73f0-4d86-ae5c-dc19a710059a">

Reply:
without some space, the user may feel that some part of the content is being cut off